asab/application.py

```diff
@@ -104,6 +108,44 @@ def handler(type):
 
 		L.info("Initializing ...")
 
+	def _is_docker(self):
+		return (
+			os.path.exists('/.dockerenv') or
+			(os.path.isfile('/proc/self/cgroup') and any('docker' in line for line in open('/proc/self/cgroup')))
+		)
+
+	def load_hostname(self):
+		hostname = platform.node()
+
+		if self._is_docker():
```

Comment:
```
remote_api = Config.get("general", "docker_remote_api")
```

should be not None ... basically it means that you have to configure "docker_remote_api" AND run in Docker

that warning
"'docker_remote_api' in 'general' was not specified. Using container ID as hostname." is not needed.